Big update (ui rewrite, unicode update ect)

ell1010 opened the pull request:
Updated and fixed all the submodules (using version 16 of unicode spec, 2.0.1 wpf-notifyicon, fixed reference to libx11 and xorgproto) reworked notification icon a lil to use more of wpf-notifyicon framework, IT NOW UPDATES WHEN YOU CHANGE COMPOSE KEY (and now the compose rules settings too) INSTEAD OF ON APPLICATION LOAD.
Updated theming to Windows 11 style with live dark mode switch
Updated framework to .NET 4.8.1
pull requests included #509  #519  #499
issues fixed #508 #495 #530

(Reposted cause the pull request got closed when i switched the branch from master to main to go with the new branch naming policies)

ell1010 committed:
test

ell1010 committed:
update

ell1010 committed:
Update README.md

ell1010 committed:
update

ell1010 committed:
update

ell1010 committed:
Update .gitmodules

ell1010 committed:
Update inserticons

ell1010 committed:
Update README.md

ell1010 committed:
switched to newest version of wpf-notifyicon and innosetup; trimmed/ reworked the code behind the icon so it uses proper win10/11 styling and it NOW UPDATES WHEN YOU CHANGE COMPOSE KEY

ell1010 committed:
updated include files and remove a hello, world dupe

ell1010 committed:
submodule update

ell1010 committed:
updated po files and builder script

ell1010 committed:
switched to dotnet-gitversion for makefile and installer.csproj, updated language gitversion build

ell1010 committed:
forked xcompose repo with added sequences (+490) from pull requests, sequences options now reload tooltip

ell1010 committed:
.net upgrade and darkmode

Upgrades to .net 4.8.1 due to 4.7.2 no longer being supported
Updated theming to use Fluent design (from Wpf-Ui by wpfui.lepo.co) with dark mode and live theme switching

ell1010 committed:
Merge branch 'samhocevar:master' into master

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

ell1010 committed:
changes to makefile so it can run properly on windows (with gnu-make from chocolatey))

ell1010 committed:
Merge branch 'master' of https://github.com/ell1010/wincompose

ell1010 committed:
Merge branch 'samhocevar:master' into main

ell1010 committed:
language files update and makefile cleanup/refinement

ell1010 committed:
Merge branch 'master' into main

ell1010 committed:
Update shot1.png

ell1010 committed:
Update xcompose

Manukineko commented:
Hello,
I have installed your release 0.9.14:
I got that error (and then a crash) when I want to change the compose key.
![image](https://github.com/user-attachments/assets/8aaecfd1-0bab-48a7-ba56-434be13e07bf)


Also, do you think you could fix #299 #320 ?

ell1010 commented:
> Hello, I have installed your release 0.9.14: I got that error (and then a crash) when I want to change the compose key. ![image](https://private-user-images.githubusercontent.com/43104858/428276258-8aaecfd1-0bab-48a7-ba56-434be13e07bf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMyNTY1NTAsIm5iZiI6MTc0MzI1NjI1MCwicGF0aCI6Ii80MzEwNDg1OC80MjgyNzYyNTgtOGFhZWNmZDEtMGJhYi00OGE3LWJhNTYtNDM0YmUxM2UwN2JmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzI5VDEzNTA1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNkNzBkYThlNDU1OGYwMjBhM2ZkODM3MzdlMjMxNDkwMDg3YzRhNzEwZjIzNzQ2N2ExMDdlZDkwYWNhYjQ1NmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.iT3_zzPVdFpgYI6XWwV0bf-6vSICCg82sKZT9y12Dp0)
> 
> Also, do you think you could fix #299 #320 ?

ive fixed the bug in the image with a hotfix, ill get the protable version uploaded later and the updated project files. Ill have a look into the other issue you linked

ell1010 committed:
fix the compose key selector pop up

ell1010 committed:
Merge branch 'master' of https://github.com/samhocevar/wincompose

ell1010 committed:
Update README.md

changed issue link to this reposity

Darthagnon commented:
Will it still work on older releases of Windows since its using .NET 4.8.1 ? I have previoiusly had issues running .NET 4.8.1 stuff on Win10

ell1010 commented:
it should do, older versions of net4 arent supported anymore, if there any problems feel free to report on my repo and ill see what i can do

ell1010 committed:
added wpf-ui license

ell1010 committed:
allowed sequence window to be resized and now shows a scrollbar when items get cut off

will probably see if theres a way to shrink the key icons (at least somewhat) when they would otherwise not fit the screen but not sure if thats possible

ell1010 committed:
translation update

ell1010 committed:
Merge branch 'samhocevar:master' into main

ell1010 committed:
Merge branch 'samhocevar:master' into main

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

ell1010 committed:
Update README.md

JorisDebonnet committed:
Updated the "download latest" line to 0.9.15.1

ell1010 committed:
Merge pull request #15 from JorisDebonnet/patch-1

Updated the "download latest" line to 0.9.15.1
